meal.go

```diff
@@ -93,21 +94,25 @@ func GetMealIds(c *gin.Context) {
 	})
 }
 
+// 실제 식단 데이터
```

Comment:
Codacy found an issue: [comment on exported type Diet should be of the form "Diet ..." (with optional leading article)](https://app.codacy.com/gh/s-owl/skhuinfo-api/pullRequest?prid=6091160)